ts/features/pn/utils/index.ts

```diff
@@ -1,48 +1,45 @@
 import I18n from "../../../i18n";
-import {
-  NotificationStatus,
-  NotificationStatusEnum
-} from "../store/types/types";
+import { NotificationStatus } from "../store/types/types";
 
 export function getNotificationStatusInfo(status: NotificationStatus): {
   label: string;
 } {
   switch (status) {
-    case NotificationStatusEnum.DELIVERED:
+    case "DELIVERED":
```

Comment:
Since we don't have anymore definitions for the list of value on `status` we could avoid the switch and convert the status to readable string link this:

```
I18n.t(`features.pn.details.timeline.status.${status}`, { defaultValue: status })
```

In this way status value would be converted in a readable string if the value received is defined in the app locales otherwise the displayed string would be the status itself.

what do you think?

Reply:
Thanks for sharing. I'll try...

Reply:
Fixed a9eb5d7b212c48ef4d6e2752176293e20ac6820b